src/ontology/mondo-edit.obo

```diff
@@ -258219,13 +258238,15 @@ xref: Orphanet:370980 {source="MONDO:subClassOf", source="OMIM:613152"}
 xref: UMLS:C2751052 {source="NCBI:mim2gene_medline", source="MONDO:equivalentTo", source="OMIM:613152"}
 is_a: MONDO:0000172 {source="OMIM:613152"} ! muscular dystrophy-dystroglycanopathy, type B
 is_a: MONDO:0018279 {source="ORDO:370980/btnt"} ! congenital muscular dystrophy without intellectual disability
+is_a: MONDO:0700067 {source="https://clinicalgenome.org/affiliation/50061/"} ! myopathy caused by mutation in FKTN
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/3622 {source="mim2gene_medgen"} ! FKTN
 property_value: confidence "1.8985054133659522" xsd:double
 
 [Term]
 id: MONDO:0013157
 name: muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type A5
 synonym: "MDDGA5" RELATED ABBREVIATION [MONDO:Lexical, OMIM:613153]
+synonym: "muscle-eye-brain-FKRP related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/